ColorHighlighter.py

```diff
@@ -1229,33 +1233,48 @@ def __init__(self, finder):
 
     def get_vars(self, view):
         color_vars_files = []
-        if is_st3():
+        if not is_st3():
+            project_settings = build_system_hack.get_project_settings(sublime.active_window())
```

Comment:
It seems to me that this can be only executed once and cached in a global variable. I'm not quite sure, though.